package.json

```diff
@@ -23,10 +23,10 @@
     "highlight"
   ],
   "license": "Unlicense",
-  "repository": "https://github.com/reed-tf2/language-tf2-cfg",
+  "repository": "https://github.com/XcapeEst/language-tf2-cfg",
```

Comment:
If you want to just use your fork, I can deprecate mine and point link people to yours. Otherwise, I can add you as a contributor to the repo and give you write access. Let me know which you prefer?

Reply:
I think it's better to add me as a contributor, that way I think we don't have to change things around with the vscode plugins.